workflows/differentialabundance.nf

```diff
@@ -92,9 +92,9 @@ if (params.study_type == 'rnaseq') {
 // Use 'limma' for specific study types or RNAseq with limma flag; else 'deseq2'
 // Also set fold change and q-value thresholds.
 tools_differential = [
-    method        : ((params.study_type in ['affy_array', 'geo_soft_file', 'maxquant']) ||
-                    (params.study_type == 'rnaseq' && params.differential_use_limma))
-                    ? 'limma' : 'deseq2',
+    method: (params.study_type == 'rnaseq' && params.differential_use_dream) ? 'dream' :
```

Comment:
Again, this logic gets out of hand as we add new methods, we can shift to a simpler approach in the near future, but this is consistent with the current logic.